wireless_modules/common_sensors/mq135_sensor.py

```diff
@@ -0,0 +1,85 @@
+from MQ135 import MQ135
+
+
+class co2:
+    def __init__(self, pin):
```

Comment:
I think this class is a bit dependant on too many classes (`MQ135` and a `DHT` instance). What happens if we decide to use a completely new CO2 sensor? We would need to replace every instance of `mq135 ` inside this class which would be a hassle!

One way we could refactor this is to have a `CO2` class that takes in the mq135 co2 sensor class and then the mq135 co2 sensor takes in a temp/humidity sensor which are all a concrete implementation of the `Sensor` ABC suggested by @hallgchris here: https://github.com/monash-human-power/data-acquisition-system/pull/61/files#r483881011. That way, we can neatly swap between different CO2 and temp/humidity sensors.

```python
class DHTSensor(Sensor):
	def __init__(self):
		...

	def read(self):
		...
class Mq135Co2Sensor(Sensor):
	def  __init__(self, temp_sensor):
		...
	def read(self):
		...
class CO2(Sensor):

	def __init__(self, co2_sensor):
		...

	def read(self):
		...
```

LMK what you think. @hallgchris you too. (pls rename class names to whatever suits you)

Reply:
Sorry if I'm missing something here, but what functionality does the `CO2` class have here? Won't it just call `read` on `Mq135Co2Sensor`, making it redundant? (I'm sure that's not the case haha)

Reply:
> Sorry if I'm missing something here, but what functionality does the CO2 class have here? Won't it just call read on Mq135Co2Sensor, making it redundant? (I'm sure that's not the case haha)

@hallgchris yeah, you're right. I wanted a 'parent' class in which all CO2 sensors are under that way we can quickly swap the instance for another 'CO2' sensor. However, since it is a `Sensor` class already, it is a bit redundant.

Reply:
I feel like if we do use a different CO2 sensor in the future, we could just write another class for it and remove the current `CO2` class. The benefit of having a general parent `CO2` class, as I see it, is that if we use a different sensor in the future we can simply make changes here and won't have to change any other files, but in the current implementation even if we do make a new class for a new sensor we only need to change it in main.py (and I highly doubt that that the current `CO2` sensor will be used in any other files even in the future). LMK if I'm getting this wrong.